portality/notifications/application_emails.py

```diff
@@ -135,6 +138,7 @@ def send_publisher_application_editor_assigned_email(application):
     if owner is not None:
         send_list.append(
             {
+                "owner" : owner,
                 "name" : owner.name,
```

Comment:
I cannot see the value "name" used anywhere further, is that needed?

Reply:
It's not, I don't think, this code needs a more general tidy up. I decided to limit myself to just adding the thing I needed rather than getting dragged into even more upstream refactoring.  Propose to leave this as is, and we'll review it again when we move to the notifications system proper.